packages/cloudapi-gql/src/schema/mutations/machines.js

```diff
@@ -189,10 +189,10 @@ module.exports.auditMachine = {
       type: new GraphQLNonNull(GraphQLID),
       description: 'The machine id'
     }
-  },
-  resolve: (root, args) => {
-    return api.machines.destroy(args.id);
   }
+  // resolve: (root, args) => {
```

Comment:
Why commented and not deleted ?

Reply:
Because we still need to add that resolver

Reply:
Please add todo then so that it throws warning in eslint then :)